docker/Dockerfile

```diff
@@ -14,15 +14,15 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-ARG BASE=zilliqa/scilla:v0.7.0
+ARG BASE=zilliqa/scilla:scilla_ubuntu_upgrade_b25539d
```

Comment:
why not tag this image more readable, like give it a version number

Reply:
that's cause we're testing a pre-release build for scilla instead of a officially released build

Reply:
so we tagged the scilla image with the hash so that we can easily reference which commit it's from, but in a proper release, it'll revert to the proper tagging